bmwqemu.pm

```diff
@@ -213,13 +213,13 @@ sub log_call (@args) {
     else {
         # key/value pairs
         my @result;
-        my $should_hide = grep { ($_ // '') eq 'secret' } @args;
+        my %args = @_;
         while (my ($key, $value) = splice(@args, 0, 2)) {
```

Comment:
```suggestion
        my %args = @_;
        while (my ($key, $value) = each %args) {
```

Reply:
That would lose the original order, that might not be wanted. Additionally, it would be random. At least it should be sorted, so then I would suggest to use `sort keys %args`.
Maybe just keep it like it is.

Reply:
yes, I didn't thought about the order, good catch!